include/dpp/message.h

```diff
@@ -587,28 +587,127 @@ struct attachment {
 	~attachment() = default;
 };
 
+enum sticker_type : uint8_t {
+	/// Nitro pack sticker
+	st_standard = 1,
+	/// Guild sticker
+	st_guild = 2
+};
+
 /**
  * @brief The file format (png, apng, lottie) of a sticker
  */
-enum sticker_type {
-	/// PNG sticker type
-	sticker_png = 1,
-	/// APNG sticker type
-	sticker_apng = 2,
-	/// LOTTIE sticker type
-	sticker_lottie = 3
+enum sticker_format : uint8_t {
+	sf_png = 1,
+	sf_apng = 2,
+	sf_lottie = 3
 };
 
 /**
  * @brief Represents stickers received in messages
  */
 struct sticker {
-	/** The ID of the sticker */
-	dpp::snowflake id;
+	/** @brief The ID of the sticker
+	 */
+	snowflake	id;
+	/** Optional: for standard stickers, id of the pack the sticker is from
+	 */
+	snowflake	pack_id;
 	/** The name of the sticker */
-	std::string name;
+	std::string	name;
+	/// description of the sticker (may be empty)
+	std::string     description;    
+	/** for guild stickers, the Discord name of a unicode emoji representing the sticker's expression.
+	 * for standard stickers, a comma-separated list of related expressions.
+	 */
+	std::string     tags;
+	/// Deprecated previously the sticker asset hash, now an empty string
```

Comment:
Do we need to support this if it’s always an empty string?

Reply:
if its in the spec on discord api i'd say so. we'll only need to add it later when or if they decide to put a value in it